### ISSUE TITLE
decimal number at start of line - error message

### ISSUE THREAD

**Issue by tallforasmurf** [wf49670/ppgen#87]
Probably doesn't come up much but I have a book with the following which produces ERROR: unknown dot command.
   
        A galvanometer shunt bears a definite ratio to the resistance
        of the galvanometer, being usually adjusted so that only .1,
    >> .01, or .001 part of the current passes through the galvanometer.

Easy to work around but perhaps you could give a pass to `^\.\d+` lines?

**Comment by wf49670** [comment]
On 8/20/2015 3:05 PM, Nat Picker wrote:
>
> Probably doesn't come up much but I have a book with the following 
> which produces ERROR: unknown dot command.
>
> |A galvanometer shunt bears a definite ratio to the resistance of the 
> galvanometer, being usually adjusted so that only .1, >> .01, or .001 
> part of the current passes through the galvanometer. |
>
> Easy to work around but perhaps you could give a pass to |^\.\d+| lines?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/wf49670/ppgen/issues/87>.
>

Thanks for the report.

I'm not sure that's the right fix, though I'll consider it. An 
alternative (which also requires a code change) would be to allow use of 
\ to escape the "." character.

I'm not sure which of those is more appropriate, but either would work. 
Your suggestion wouldn't require any work on the part of the PPer, but 
if there are other situations that could have a "." in position 1 then 
the \ workaround might be necessary for those, anyway.

**Comment by wf49670** [comment]
Fixed in current development release, 3.52j.

I decided to go ahead and accept . in position 1 if stand-alone or followed by a character other than a-z

**Comment by tallforasmurf** [comment]
Nothing like a year-end tidy-up! I take it ppgen is still a live app?

**Comment by wf49670** [comment]
Yes, though development has been moribund for a couple of years.
